tests/Integration/Infrastructure/Controller/Regulation/ExportRegulationControllerTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Integration\Infrastructure\Controller\Regulation;
+
+use App\Tests\Integration\Infrastructure\Controller\AbstractWebTestCase;
+
+final class ExportRegulationControllerTest extends AbstractWebTestCase
+{
+    public function testDownload(): void
+    {
+        $client = $this->login();
+        $client->request('GET', '/regulations/e413a47e-5928-4353-a8b2-8b7dda27f9a5/export');
```

Comment:
Ça doit fonctionner car on a `defaults: _format => docx`, mais ne vaut-il pas mieux une URL explicite ?

```suggestion
        $client->request('GET', '/regulations/e413a47e-5928-4353-a8b2-8b7dda27f9a5/export.docx');
```